Makefile

```diff
@@ -102,7 +102,9 @@ generate:
 .PHONY: generate-swagger
 generate-swagger:
 	@hash swagger > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
-		$(GO) get -u github.com/go-swagger/go-swagger/cmd/swagger; \
+		export GO111MODULE="on"; \
```

Comment:
It may be a better idea to only set `GO111MODULE="on"` only for the command itself like so:
```
GO111MODULE="on" $(GO) get -u github.com/go-swagger/go-swagger/cmd/swagger@latest; \
```

Reply:
Ah damn missed that. @kolaente is correct here. Do not `export GO111MODULE`.

Reply:
```suggestion
```